chapters/functions.tex

```diff
@@ -1880,10 +1880,11 @@ \section{Function Inlining and Event Generation}\label{function-inlining-and-eve
 "GenerateEvents" "=" ( false | true )
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
-Has only an effect within a function declaration
-
-If \lstinline!GenerateEvents = true!, the model developer proposes that crossing functions in the function should generate events (one possibility of doing this is to inline the function and generate events for the inlined function).
+Has only an effect within a function declaration.
 
+If \lstinline!GenerateEvents = true!, the model developer proposes that crossing functions in the function should generate events.
+As defaults this implies \lstinline!Inline = true!, but it can be overriden by specifying one of the inline-annotations (with true value).
+Tools should generate diagnostics if it is not possible to generate events for crossing functions in the function.
```

Comment:
Trying to formulate in a more familiar (but stronger than usual) way:
```suggestion
As event-generation can be crucial for correct simulation, a diagnostic is strongly recommended in case events will not be generated for a function with `GenerateEvents = true`.
```

Alternatively, wouldn't it be better if we took the opportunity to remove the tool's freedom to choose whether to generate events or not?